src/ILCompiler.Compiler/src/Compiler/DependencyAnalysis/GCStaticsNode.cs

```diff
@@ -73,11 +84,22 @@ public override ObjectData GetData(NodeFactory factory, bool relocsOnly = false)
 
             if (factory.Target.Abi == TargetAbi.CoreRT)
             {
-                builder.EmitPointerReloc(GetGCStaticEETypeNode(factory), 1);
+                int delta = 1;
```

Comment:
Oh, yeah, that was backwards. Changed to Uninitialized and switched to use GCStaticRegionConstants.